type.go

```diff
@@ -70,6 +78,7 @@ type field struct {
 var (
 	ErrUnmarshal        = errgo.New("httprequest unmarshal error")
 	ErrBadUnmarshalType = errgo.New("httprequest bad unmarshal type")
+	emptyValue          = reflect.Value{}
```

Comment:
I don't think you need this, but for future reference, the IsValid method is a better way to do this.